sdk/src/user.ts

```diff
@@ -1447,30 +1467,60 @@ export class User {
 	 */
 	public getMaxTradeSizeUSDC(
```

Comment:
feel like we should similarly fork this function at the top into `getMaxTradeSizeUSDCForSpot` and `getMaxTradeSizeUSDCForPerp` and then use the liability weight for the spot one